### ISSUE TITLE
relabel MONDO_0010107

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#1447]
**Mondo term (ID and Label):**
MONDO_0010107

**Suggested new label:**
46,XY SEX REVERSAL 11

**_Optional_: Any additional information (like supporting evidence, PubMed ID, etc.)**
OMIM updated this name
https://www.omim.org/entry/273250#description

cc @maglott

**Comment by cmungall** [comment]
Let's be very careful here.

Are we totally sure that for our current definition of TRS, the gene is DHX37?

When we see a general-sounding class relabeled as a more specific-sounding one, is this really just a chance of label or should we split the concept?

Can you summarize some of the literature?

**Comment by maglott** [comment]
Good point.  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6944638/  leaves the door open for the involvement of other genes.
"We provide compelling genetic evidence that specific variants in the DEAH-box protein DHX37, which encodes a putative RNA helicase, are a frequent cause of nonsyndromic 46,XY gonadal dysgenesis as well as TRS. "

**Comment by nicolevasilevsky** [comment]
Should we split these terms then?

It doesn't look like [Orphanet](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=1300&Disease_Disease_Search_diseaseGroup=983&Disease_Disease_Search_diseaseType=ORPHA&Disease(s)/group%20of%20diseases=Testicular-regression-syndrome&title=Testicular%20regression%20syndrome&search=Disease_Search_Simple) has any genes associated with Testicular regression syndrome

**Comment by nicolevasilevsky** [comment]
Per call, yes split
